docs/guidelines.md

```diff
@@ -1,2 +1,33 @@
-# Basic Guidelines to follow while integration
+# Guidelines to follow while integration
+This section captures the guidelines for broad use cases which relates to almost all of our Ultra partners’ applications.
 
+### General
+These general guidelines apply across all the pages within a partner’s application.
+
+- No external redirection in your app.
+- We do not tolerate obstruction of Flipkart’s property within Ultra.
```

Comment:
I know this was present in the original documentation, however, I think sounds harsh. Should we change this to something like: " No obstruction of Flipkart's property within Ultra"

Reply:
Done. Please check the other commit of the same branch (OjesviPandita-Upd-Guidelines).